test/e2e/atlas/helper_test.go

```diff
@@ -770,3 +776,47 @@ func compareStingsWithHiddenPart(expectedSting, actualString string, specialChar
 	}
 	return true
 }
+
+// createJSONFile creates a new JSON file at the specified path with the specified data
+// and also registers its deletion on test cleanup.
+func createJSONFile(t *testing.T, data interface{}, path string) {
+	t.Helper()
+	jsonData, err := json.Marshal(data)
+	if err != nil {
+		t.Errorf("Error marshaling to JSON: %v", err)
+		return
+	}
+
+	err = os.WriteFile(path, jsonData, 0600)
+	if err != nil {
+		t.Errorf("Error writing JSON to file: %v", err)
+		return
+	}
+
+	t.Cleanup(func() {
+		if err := os.Remove(path); err != nil {
+			t.Errorf("Error deleting file: %v", err)
+		}
+	})
+}
+
+func enableCompliancePolicy(projectID string) error {
+	cliPath, err := e2e.AtlasCLIBin()
+	if err != nil {
+		return fmt.Errorf("%w: invalid bin", err)
+	}
+	cmd := exec.Command(cliPath,
+		backupsEntity,
+		compliancepolicyEntity,
+		"enable",
+		"--projectId",
+		projectID,
+		"--authorizedEmail",
+		authorizedEmail,
+		"-o=json",
+		"--watch", // avoiding HTTP 400 Bad Request "CANNOT_UPDATE_BACKUP_COMPLIANCE_POLICY_SETTINGS_WITH_PENDING_ACTION".
+	)
+	cmd.Env = os.Environ()
+	_, outputErr := cmd.CombinedOutput()
+	return fmt.Errorf("executing command unsuccessful: %w", outputErr)
```

Comment:
```suggestion
	output, outputErr := cmd.CombinedOutput()
	if outputErr != nil {
		return fmt.Errorf("%w\n %s", outputErr, string(output))
	}
	return nil
```

Reply:
Realised this myself 2 minutes ago, had hard tunnel vision